packages/host/app/services/store.ts

```diff
@@ -272,9 +316,27 @@ export default class StoreService extends Service {
     }
 
     if (maybeUpdatedCard?.id) {
+      if (!this.cardApiCache) {
+        this.cardApiCache = await this.cardService.getAPI();
+      }
```

Comment:
no need to cache this--it's actually cached in the loader, you can just get it each time that you need it.

Reply:
It's cached so that it can be reused in the unloadResource function. Otherwise, unloadResource would have to be an async function, but we need it to be synchronous for use in registerDestructor. Do you have any other idea how to get cardAPI in unreloadResource?